drainer/handler.py

```diff
@@ -4,62 +4,159 @@
 import os.path
 import re
 import yaml
+import json
+import time
 
 from botocore.signers import RequestSigner
 import kubernetes as k8s
 from kubernetes.client.rest import ApiException
 
-from k8s_utils import (abandon_lifecycle_action, cordon_node, node_exists, remove_all_pods)
+from k8s_utils import (
+    abandon_lifecycle_action,
+    cordon_node,
+    node_exists,
+    remove_all_pods,
+)
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
-KUBE_FILEPATH = '/tmp/kubeconfig'
-CLUSTER_NAME = os.environ['CLUSTER_NAME']
-REGION = os.environ['AWS_REGION']
+KUBE_FILEPATH = "/tmp/kubeconfig"
+CLUSTER_NAME = os.environ["CLUSTER_NAME"]
+REGION = os.environ["AWS_REGION"]
+SQS_QUEUE_URL = os.environ["SQS_QUEUE_URL"]
+CFN_STACK_NAME = os.environ["CFN_STACK_NAME"]
+CFN_RESOURCE_ID = os.environ["CFN_RESOURCE_ID"]
 
-eks = boto3.client('eks', region_name=REGION)
-ec2 = boto3.client('ec2', region_name=REGION)
-asg = boto3.client('autoscaling', region_name=REGION)
+eks = boto3.client("eks", region_name=REGION)
+ec2 = boto3.client("ec2", region_name=REGION)
+asg = boto3.client("autoscaling", region_name=REGION)
+sqs = boto3.client("sqs", region_name=REGION)
+cfn = boto3.client("cloudformation", region_name=REGION)
+
+
+def proccess_queue():
```

Comment:
```suggestion
def process_queue():
```

Reply:
We should probably make it CI/CD and have the TF code pulling the zip to deploy it from git.